### PR TITLE
Add conversion functions.

### PR DESCRIPTION
**Pull request by bitbckt**

### CODE REVIEW

File: faunadb/query.py [hunk 1]
```diff
@@ -557,6 +557,18 @@ def not_(boolean):
   """See the `docs <https://fauna.com/documentation/queries#misc_functions>`__."""
   return _fn({"not": boolean})
 
+def to_string(expr):
+  return _fn({"to_string": expr})
```
Comment (erickpintor) [inline faunadb/query.py]: Do we have a section in the docs that we can link here?

Comment (bitbckt) [reply faunadb/query.py]: Not yet.

**Comment by marrony** [comment]
yes, `test_query.py`

**Comment by bitbckt** [comment]
Hmm. I don't see `@date` support in this driver to test `to_date`.

**Pull request merged**